src/mbgl/renderer/render_layer.hpp

```diff
@@ -101,6 +104,12 @@ class RenderLayer {
     // in the console to inform the developer.
     void checkRenderability(const PaintParameters&, uint32_t activeBindingCount);
 
+    // Code specific to RenderTiles sorting / filtering
+    virtual RenderTiles filterRenderTiles(RenderTiles) const;
+    virtual void sortRenderTiles(const TransformState&);
+    RenderTiles filterRenderTiles(RenderTiles, const std::function<bool (RenderTile&)>&) const;
```

Comment:
We could use a function pointer here instead of `std::function` to avoid generating `std::function` objects.

Reply:
Done